apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/PageController.java

```diff
@@ -93,6 +101,12 @@ public PageCreateResponse createPage(@RequestBody PageCreateRequest request) {
         Long userId = userDetailsProvider.getCurrentUserDetails().getId();
         return creator.createPage(request, userId);
     }
+    
+    @GetMapping("{id}/details")
+    public PageDetailResponse.PagedDetail getPageDetails(@PathVariable("id") Long pageId) {
+    	return pageFinder.getPageDetails(pageId);
+    }
```

Comment:
Issue (blocking): I believe this API was removed in a separate PR but was added back in merging with yours. I'm fairly sure this can be removed